SDWebImage/Core/SDImageCacheConfig.m

```diff
@@ -36,7 +36,11 @@ - (instancetype)init {
         _maxDiskSize = 0;
         _diskCacheExpireType = SDImageCacheConfigExpireTypeModificationDate;
         _fileManager = nil;
-        _ioQueueAttributes = DISPATCH_QUEUE_SERIAL; // NULL
+        if (@available(iOS 10.0, *)) {
```

Comment:
Need all Apple platforms in available expression. At least 4 (tvOS/watchOS/macOS), visionOS use *

Reply:
From the documentation: 

iOS 10.0+
Mac Catalyst 13.1+
macOS 10.12+
tvOS 10.0+
visionOS 1.0+Beta
watchOS 3.0+

Reply:
Ops, i will fix this in next commit. (I forget all platforms...)